File: scripts/testes-outras-linguagens/ProblemaA/Testes-PHP/TimeLimitedExcepted.php
<?php

// Lê a linha de entrada
$input_line = trim(fgets(STDIN));
$notes = explode(' ', $input_line);

$P = (float)$notes[0];
$AP = (float)$notes[1];
$TB = (float)$notes[2];

// --- LOOP INFINITO INTENCIONAL ---
// O programa ficará preso aqui, excedendo o tempo limite (TLE).
while (true) {
    // Incrementa uma variável para que o loop não seja otimizado, 
    // mas o programa nunca sairá daqui.
    $i++; 
}

// O código correto abaixo nunca será alcançado
$A1 = ($P * 7.0 + $AP * 2.0 + $TB * 1.0) / 10.0;
echo number_format($A1, 3, '.', '') . "\n";
?>

File: scripts/testes-outras-linguagens/ProblemaA/TestesC#/TimeLimitedExcepted.cs
using System;
using System.Linq;

public class Program
{
    public static void Main(string[] args)
    {
        string[] input = Console.ReadLine().Split(' ');
        
        float P = float.Parse(input[0]);
        float AP = float.Parse(input[1]);
        float TB = float.Parse(input[2]);
        
        // Cálculo correto
        double A1 = (P * 7.0 + AP * 2.0 + TB * 1.0) / 10.0;
        
        // --- LOOP INFINITO INTENCIONAL ---
        // O programa fica preso neste loop, excedendo o tempo limite (TLE).
        while (true)
        {
            // Operação vazia para garantir que o loop não termine.
        }
        
        // A saída nunca será alcançada.
        Console.Write(A1.ToString("F3"));
    }
}